python/event_generator.hpp

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <arbor/common_types.hpp>
+#include <arbor/event_generator.hpp>
+#include <arbor/schedule.hpp>
+
```

Comment:
I don't think we need this header. All of the contents of this file can go into the `event_generators.cpp`.
The `X_schedule_shim` types don't need to be used anywhere else that I can see.

Reply:
Done.